linode/manager
# upcoming: [M3-8012] - Fix PG event text formatting

## Description 📝
This PR updates the types and constants for action events for Placement Groups. Specifically, the action events are updated with a present-tense and imperative-style for consistency with other entity events. These changes resolve issues with Placement Group events missing the link and username.

## Changes  🔄
List any change relevant to the reviewer.
- Remove the `d` and `ed` from events to make them consistent with other entity action events.
(Ex. `placement_group_created` becomes `placement_group_create` and `placement_group_unassigned` becomes `placement_group_unassign`.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![before-aaa](https://github.com/linode/manager/assets/119514965/6c1fe83d-0c46-4d60-a1cf-b49f514c2cda) | ![after-bb](https://github.com/linode/manager/assets/119514965/6495feda-1eeb-4ff3-8423-ebbea3afa184) |


## How to test 🧪
### Prerequisites
(How to setup test environment)
- Have the `placement-group` customer tag on the user account used for testing.
- Using the Cloud Manager Dev Tools:
  - Enable the `Placement Groups` feature flag
  - Switch to `Dev` environment
  - Disable the MSW

### Verification steps
(How to verify changes)
- Verify the following events emit an event that is formatted properly (with link and username).
  - `placement_group_create`
  - `placement_group_assign`
  - `placement_group_unassign`
  - `placement_group_update`
  - `placement_group_delete`


## As an Author I have considered 🤔
*Check all that apply*
